tests/test_georaster.py

```diff
@@ -121,9 +131,17 @@ def test_copy(self):
             assert r.__getattribute__(attr) == r2.__getattribute__(attr)
 
         # Check data array
-        assert np.count_nonzero(~r.data == r2.data) == 0
+        assert np.all(r.data == r2.data)
```

Comment:
This  won't work with NaNs, as previously mentioned. The assertion below is safer
```python
assert np.array_equal(r.data, r2.data, nan_equal=True)
```

Reply:
good to know

Reply:
Amaury I let you change those, I'll do the same in `test_satimg` and `test_dem`

Reply:
@adehecq actually let me change those everywhere, I'll squash and merge